docs/developer/solutions/deploy.rst

```diff
@@ -0,0 +1,23 @@
+Deployment And Experimenting
```

Comment:
Weird to only have the second word ending in `-ing`.

I would expect something like `Deploying and Experimenting`.